src/opf/socwr.jl

```diff
@@ -1,3 +1,6 @@
+mutable struct SOCOPFQuad <: Formulation end
+mutable struct SOCOPF <: Formulation end
```

Comment:
No need for `mutable` since we do not have any attributes
```suggestion
struct SOCOPFQuad <: AbstractFormulation end
struct SOCOPF <: AbstractFormulation end
```